db/migrate/20150612135546_create_host_status.rb

```diff
@@ -10,7 +10,7 @@ def up
     add_foreign_key "host_status", "hosts", :name => "host_status_hosts_host_id_fk", :column => 'host_id'
     add_column :hosts, :global_status, :integer, :default => 0, :null => false
 
-    Host.all.each do |host|
+    Host.includes(:host_statuses, :last_report_object).find_each do |host|
```

Comment:
`find_in_batches` is used inside `find_each`, it's just a wrapper if you need to work with the whole batch IIUC